Bottom: 805703fe3d7aa3ae48948f376b010e84f83023c6
Top:    c99fe78f750fa7245be2932f3362537c52546b84
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 18:01:15 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
